raspberry/read_sbus/radioDataParser.py

```diff
@@ -0,0 +1,122 @@
+
+
+# Maximum pitch value corresponding to the top position of the stick (30deg = 0.52rad)
+PITCH_MAX_RAD = 0.5235987755982988
+
+# Value from the radio when the stick is at the lowest position
+RADIO_STICK_MIN = 300
+
+# Value from the radio when the stick is at the highest position
+RADIO_STICK_MAX = 1700
+
+# Maximum yaw rate (rad/s) 10 deg/s = 0.1745 rad/s
+RADIO_MAX_YAW_RATE_RAD_SEC = 0.1745
+
+# Trimmer A on RC - maximum quaternion XY gain
+RADIO_TRIMMER_MAX_QUATERNION_XY_GAIN = 100.
+
+# Trimmer B on RC - maximum quaternion Z gain
+RADIO_TRIMMER_MAX_QUATERNION_Z_GAIN = 20.
+
+# Trimmer C on RC - maximum omega xy gain
+RADIO_TRIMMER_MAX_OMEGA_XY_GAIN = 0.5
+
+# Trimmer E on RC - maximum omega z gain
+RADIO_TRIMMER_MAX_OMEGA_Z_GAIN = 0.05
+
+
+RADIO_CHANNEL_YAW_RATE = 0
+RADIO_CHANNEL_PITCH = 1
+RADIO_CHANNEL_THROTTLE = 2
+RADIO_CHANNEL_ROLL = 3
+RADIO_CHANNEL_VRA = 5
+RADIO_CHANNEL_VRB = 7
+RADIO_CHANNEL_VRC = 6
+RADIO_CHANNEL_VRE = 11
+RADIO_CHANNEL_SWITCH_A = 9
+RADIO_CHANNEL_SWITCH_B = 8
+RADIO_CHANNEL_SWITCH_C = 4
+RADIO_CHANNEL_SWITCH_D = 10
+
+DOWN = "DOWN"
```

Comment:
You can use an [enum](https://docs.python.org/3/library/enum.html) for this (same as we did in C++)

Reply:
Done.